api/envoy/extensions/load_balancing_policies/ring_hash/v3/ring_hash.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+package envoy.extensions.load_balancing_policies.ring_hash.v3;
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.load_balancing_policies.ring_hash.v3";
+option java_outer_classname = "RoundRobinProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/load_balancing_policies/ring_hash/v3;ring_hashv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Ring Hash Load Balancing Policy]
+
+// This configuration allows the built-in RING_HASH LB policy to be configured via the LB policy
+// extension point. See the :ref:`load balancing architecture overview
+// <arch_overview_load_balancing_types>` for more information.
+// [#extension: envoy.clusters.lb_policy]
+message RingHash {
+  // The hash function used to hash hosts onto the ketama ring.
+  enum HashFunction {
+    // Use `xxHash <https://github.com/Cyan4973/xxHash>`_, this is the default hash function.
+    XX_HASH = 0;
```

Comment:
Do we want a real `DEFAULT` field to allow implementation choice and avoid hard coding this forever?

Reply:
Agree.

Reply:
Done.